### DISCUSSION TITLE
Public dashboard?

### DISCUSSION THREAD

**Discussion by scriptzteam** [henrygd/beszel#101] [Ideas]
Is there any plans for Public dashboard without the need of logging-in?
So guests will be able to see only the stats.

Thanks.

**Comment by henrygd** [comment]
Maybe in the future, but I don't think it's a straightforward feature to add with the way things are set up currently.

The alternative is to create a user with a readonly role and share systems with that user. But that still requires the guest to log in.

There's also an issue for this here: #11
